### ISSUE TITLE
Round Robin doesn't seem to be working

### ISSUE THREAD

**Issue by itecedor** [ruflin/Elastica#658]
Using the example from the documentation, I have two nodes running, and am instantiating the Elastica client like this:

    $this->_Elastica    = new \Elastica\Client(array(
    'servers' => array(
      array('host' => 'localhost', 'port' => 9200),
      array('host' => 'localhost', 'port' => 9201)
    )
    ));

The node at 9200's name is "Black Fox" and the one at 9201 is "Chimera". When I run the code above as-is, and then print out $this->_Elastica->getStatus()->getServerStatus(), it is always:

    Array
    (
    [ok] => 1
    [status] => 200
    [name] => Black Fox
    [version] => Array
        (
            [number] => 0.90.1
            [snapshot_build] =>
            [lucene_version] => 4.3
        )

    [tagline] => You Know, for Search
    )

In other words, it's always connected to "Black Fox". If I swap the order of the hosts, it always prints out "Chimera".

Am I missing something? Shouldn't they be alternating?

**Comment by im-denisenko** [comment]
Fixed via #661.

```php
$client = new Elastica\Client([
    'connectionStrategy' => 'RoundRobin',
    'connections' => [
        ['host' => 'foo'],
        ['host' => 'bar'],
    ],
]);
var_dump($client->getConnection()->getHost()); // foo
var_dump($client->getConnection()->getHost()); // bar
var_dump($client->getConnection()->getHost()); // bar
var_dump($client->getConnection()->getHost()); // foo
```
